gnome-initial-setup/gis-driver.c

```diff
@@ -80,12 +81,40 @@ struct _GisDriverPrivate {
   gchar *personality;
   gchar *lang_id;
 
+  gboolean is_live_session;
+
   GisDriverMode mode;
 };
 typedef struct _GisDriverPrivate GisDriverPrivate;
 
 G_DEFINE_TYPE_WITH_PRIVATE(GisDriver, gis_driver, GTK_TYPE_APPLICATION)
 
+static gboolean
+running_live_session (void)
+{
+  gboolean has_live_boot_param;
+  GError *error;
+  gchar *contents;
+
+  contents = NULL;
+  error = NULL;
+
+  g_file_get_contents ("/proc/cmdline", &contents, NULL, &error);
+
+  if (error)
```

Comment:
… but I see that many, many other places in this application use the `foo (..., &error); if (error) {...}` style so I suppose I can let this slide, even if I don't like it myself ☺